### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by markhaur**
added necessary project related information along with contributors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
 # srtmod
+
+**srtmod** is a CLI application written in [Go](https://go.dev/) and delivered as a [single executable](https://github.com/markhaur/srtmod/releases/tag/v0.1.1) to add/subtract time offset from .srt subtitle files.
```
Comment (jarri-abidi) [inline README.md]: Let's link to releases so we don't have to change this everytime.
```suggestion
**srtmod** is a CLI application written in [Go](https://go.dev/) and delivered as a [single executable](https://github.com/markhaur/srtmod/releases) to add/subtract time offset from .srt subtitle files.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+```
+or can process single .srt file using args
+```bash
+./srtmod -i=input.srt -o=output.srt -offset=-1m5s
+```
+## Contributing
+Want to contribute? Awesome! The most basic way to show your support is to star the project or to raise issues. Pull requests are highly welcome.
+
+Please make sure to update tests as appropriate.
+## Potential Maintainers
```
Comment (jarri-abidi) [inline README.md]: btw why potential? 😆 we're already maintaining it - you can count on me for pr reviews at the very least
```suggestion
## Maintainers
```

File: README.md (hunk 1 above)
Comment (markhaur) [reply README.md]: I thought about this at the first place, but went this way. Sure let me commit your suggestion @jarri-abidi

File: README.md (hunk 2 above)
Comment (markhaur) [reply README.md]: hahaha @jarri-abidi sure lemme incorporate your suggestions bro!

**Review by jarri-abidi — APPROVED** [review]

**Pull request merged**